src/Umbraco.Web.UI.Client/src/less/application/grid.less

```diff
@@ -217,19 +223,17 @@ body {
     padding-left:20px
 }
 
-
-
 @media (max-width: 767px) {
 
     // make leftcolumn smaller on tablets
     #leftcolumn {
-      width: 60px;
+        width: 61px;
     }
     #applications ul.sections {
-        width: 60px;
+        width: 61px;
     }
     ul.sections.sections-tray {
-        width: 60px;
+        width: 61px;
```

Comment:
I had reversed it for desktop and tablet, but I probably forgot it for mobile - it should just has the width as the current version.